bin/post_compile

```diff
@@ -10,7 +10,9 @@
 set -euo pipefail
 
 # Make the current Git revision accessible at <site-root>/revision.txt
-echo "$SOURCE_VERSION" > .build/revision.txt
+export REVISION=${SOURCE_VERSION:-$(git rev-parse HEAD)}
```

Comment:
Heroku's build sets the variable `SOURCE_VERSION`. In Travis we execute the Git command.